js/app/interfaces/card.js

```diff
@@ -311,7 +311,7 @@ const Card = new Lang.Interface({
     _count_set: function (set_obj, callback) {
         let query = Eknc.QueryObject.new_from_props({
             tags_match_any: set_obj.child_tags,
-            limit: -1,
+            limit: GLib.MAXUINT32,
```

Comment:
Not working, somehow this is getting converted to -1 before we create the uri for xapian-bridge. I double checked, changing it to 1000 or 10000 then it works fine.